core/db/migrate/20151026093607_change_return_item_pre_tax_amount_to_amount.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeReturnItemPreTaxAmountToAmount < ActiveRecord::Migration
+  def up
+    execute(<<-SQL)
+      UPDATE spree_return_items
+      SET pre_tax_amount = pre_tax_amount + included_tax_total;
+    SQL
```

Comment:
do we need to be worried about included_tax_total ever being nil? I'd had to nullify pre_tax_amount from included_tax_total being nil. Should we switch to WHERE pre_tax_amount is null to be safe?